testing/path.go

```diff
@@ -115,3 +115,97 @@ func (path *Path) RelayPacketWithResults(packet channeltypes.Packet) (*abci.Exec
 
 	return nil, nil, fmt.Errorf("packet commitment does not exist on either endpoint for provided packet")
 }
+
+// Setup constructs a TM client, connection, and channel on both chains provided. It will
+// fail if any error occurs. The clientID's, TestConnections, and TestChannels are returned
+// for both chains. The channels created are connected to the ibc-transfer application.
+func (path *Path) Setup() {
+	path.SetupConnections()
+
+	// channels can also be referenced through the returned connections
+	path.CreateChannels()
+}
+
+// SetupClients is a helper function to create clients on both chains. It assumes the
+// caller does not anticipate any errors.
+func (path *Path) SetupClients() {
+	err := path.EndpointA.CreateClient()
+	if err != nil {
+		panic(err)
+	}
+
+	err = path.EndpointB.CreateClient()
+	if err != nil {
+		panic(err)
+	}
+}
+
+// SetupClientConnections is a helper function to create clients and the appropriate
+// connections on both the source and counterparty chain. It assumes the caller does not
+// anticipate any errors.
+func (path *Path) SetupConnections() {
+	path.SetupClients()
+
+	path.CreateConnections()
+}
+
+// CreateConnection constructs and executes connection handshake messages in order to create
+// OPEN channels on chainA and chainB. The connection information of for chainA and chainB
+// are returned within a TestConnection struct. The function expects the connections to be
+// successfully opened otherwise testing will fail.
```

Comment:
```suggestion
// CreateConnections constructs and executes connection handshake messages in order to create
// OPEN connections on chainA and chainB. The function expects the connections to be
// successfully opened otherwise testing will fail.
```

Seems this docstring was out of date